weed/topology/topology.go

```diff
@@ -24,30 +24,24 @@ import (
 )
 
 type Topology struct {
-	vacuumLockCounter int64
 	NodeImpl
-
-	collectionMap  *util.ConcurrentReadMap
-	ecShardMap     map[needle.VolumeId]*EcShardLocations
-	ecShardMapLock sync.RWMutex
-
-	pulse int64
-
-	volumeSizeLimit  uint64
-	replicationAsMin bool
-
-	Sequence sequence.Sequencer
-
+	Sequence           sequence.Sequencer
+	RaftServer         raft.Server
+	vacuumLockCounter  int64
```

Comment:
moving it here may cause atomic operation to fail on 32bit OS.

There is only one instance of Topology in memory. No need to optimize for a few bytes.